modules/setting/setting_i18n.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
```

Comment:
The file name could be i18n.go

Reply:
And I would like these change become another PR to make the PR clear.

Reply:
It is very trivial, I even couldn't find a good name for that PR.

Since they are `setting` related, I hope it can get merged together. Otherwise it would cost too much time.